src/common/misc/RecipientsSearchModel.ts

```diff
@@ -27,7 +36,7 @@ export class RecipientsSearchModel {
 	constructor(
 		private readonly recipientsModel: RecipientsModel,
 		private readonly contactModel: ContactModel,
-		private readonly suggestionsProvider: ((query: String) => Promise<readonly ContactSuggestion[]>) | null,
+		private readonly suggestionsProvider: ContactSuggestionProvider | null,
```

Comment:
this one doesn't have to be nullable anymore as we always have one now

Reply:
yes. Removed null.